github-usercard/src/App.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import './App.css';
+import axios from 'axios'
+import UserCard from './components/UserCard'
+class App extends React.Component{
+  constructor(){
+    super()
+    this.state = {
+      profile: {},
+      username: 'ariuka11',
+      followers: []
+    }
+  }
+  componentDidMount(){
+    axios.get(`https://api.github.com/users/${this.state.username}`)
+      .then(res => {
+        console.log(res)
+        this.setState({profile: res.data})
+      })
+      .catch(err => {
+        console.log('Not success', err)
+      })
+    axios.get(`https://api.github.com/users/${this.state.username}/followers`)
+      .then(res => {
+        console.log(res)
+        this.setState({followers: res.data})
+      })
+      .catch(err => {
+        console.log('Not success', err)
+      })
+  }
+  render(){
+    return(
+      <div>
+        <UserCard {...this.state.profile} followers={this.state.followers} />
```

Comment:
Interesting way of passing all the individual properties of the user object as a prop here.  I didn't think to do it that.  I would have preferred to pass the whole object as user={this.state.profile} or just pass the 3 properties that you're using.